### ISSUE TITLE
Youtube videos embeds lack a title attribute

### ISSUE THREAD

**Issue by beckej13820** [pressbooks/pressbooks#1696]
### Description

Youtube videos that are embedded though the automated OEmbed method are missing a title attribute, which is causing it to fail some automated accessibility checkers. 

At our SUNY Training from My Blindspot and Deque accessibility consultants, they encouraged us to the the opensource Deque Chrome and Firefox plugin called aXe. This was marked a "serious" violation (which is basically "medium", below critical and blocker)

### Steps to Reproduce

1. embed a youtube video using the automated oembed
2. download Axe plugin for Chrome or Firefox
3. Analyze the page

**Expected behavior:** 

I'm not sure what the title should actually be, whether it should say "Youtube video" or try to pull the title of the video.

**Actual behavior:** 

There's no title attribute.

**Comment by gavinpressbooks** [comment]
This issue was investigated across Pressbooks pages with and without embedded video using the aXe tool. A 'serious' issue was identified by the tool with a possible cause of a lack of title attribute on both pages with and without embedded videos. The tool identified the offending section of code as being related to the svg arrow element as part of the header controls to expand the table of contents, which explains the tool is identifying it on all Pressbooks webbook pages, not only those with embedded video. Inspection of the iframe video element shows an included title attribute matching the title of the Youtube video itself. Please see attached screenshots:
![Screen Shot 2021-06-14 at 10 30 45 AM](https://user-images.githubusercontent.com/85123231/122204797-8733ad00-ce6d-11eb-8f8e-87ea40cf6b3b.png)
![Screen Shot 2021-06-14 at 10 30 53 AM](https://user-images.githubusercontent.com/85123231/122204815-8dc22480-ce6d-11eb-9c20-cf3cbb86490b.png)
![Screen Shot 2021-06-14 at 10 28 35 AM](https://user-images.githubusercontent.com/85123231/122204919-acc0b680-ce6d-11eb-96a9-6c6184b73a19.png)
![Screen Shot 2021-06-14 at 10 29 38 AM](https://user-images.githubusercontent.com/85123231/122204929-ae8a7a00-ce6d-11eb-9b4b-ae22ce17734e.png)

**Comment by SteelWagstaff** [comment]
@beckej13820 Can you let us know if you're still seeing the original problem with YouTube titles? We think this may not be a relevant issue any longer ...

**Comment by beckej13820** [comment]
Looks like it was fixed upstream:

https://core.trac.wordpress.org/ticket/40245
